### ISSUE TITLE
Nothing gets extracted from PlayStationNow.msi

### ISSUE THREAD

**Issue by mirh** [activescott/lessmsi#106]
[Sample](https://github.com/activescott/lessmsi/files/1871301/PlayStationNow.zip)

**Comment by activescott** [comment]
Thank you for the bug report. I'm pretty busy with some other projects these days but I'll accept pull requests (and relentlessly assist if you have questions or get stuck) or I am going to take some time off my day job in July and _maybe_ I'll dig into this then.

**Comment by bhattumang7** [comment]
@activescott 
I tried to do extract the same msi file attached to this bug. 4th column from query to media table (in CabsFromMsiToDisk method in code) is coming as a blank string. This is a valid scenario. The blank string is causing the loop to skip the row. I see that the [MSDN](https://docs.microsoft.com/en-us/windows/desktop/msi/media-table) says that if no cab files are used then the 4th column can be empty. I also see that the Binary table is having some data. I think we should extract it and show it to the user. 

Also, the binary table does not match with what the lessmsi tool shown in UI upon MSI load so we might want to change that as well. Let me know your thoughts. I can take this forward if you provide directions.

**Comment by activescott** [comment]
@bhattumang7 I'm not sure where the embedded files are for this particular installer. More information about where files are stored in MSI files is at https://docs.microsoft.com/en-us/windows/desktop/Msi/ordering-file-sequence-numbers-in-a-cabinet-file-table-and-media-table

As I understand it the Binary table wouldn't normally be used for installable files.

If you can get this msi to extract the proper files and all the existing tests continue to work that would be great.  

If you have a more specific question let me know and I'll do my best to help!

**Comment by jjurkus** [comment]
Where could a complete version of this msi file be found? This seems to be missing a cab file. Anyway, newer versions seems to be packaged differently.

**Comment by mirh** [comment]
That comes from here https://download-psnow.playstation.com/downloads/psnow/pc/PlayStationNow-9.7.15.exe
But there was no change in packaging. It's just that you need to use the `/extract` argument to get it.

**Comment by jjurkus** [comment]
Right, thanks. I think this would be on par with one of those autodesk msi files, where the files are supplied the same way. IE: the files are all there (after using the /extract argument), so at least they are available.

**Comment by mirh** [comment]
Actually, I just found out they aren't. 
You can find file sixaxis.inf in your %TEMP% folder when you run the installer normally for example. 
But be it in the .msi, or from the other files you get with /extract, that's nowhere to be seen instead.
